dnsprovider/pkg/dnsprovider/providers/aws/route53/route53.go

```diff
@@ -63,7 +63,11 @@ func newRoute53(config io.Reader) (*Interface, error) {
 	// e.g. https://github.com/kubernetes/kops/issues/605
 	awsConfig = awsConfig.WithCredentialsChainVerboseErrors(true)
 
-	svc := route53.New(session.New(), awsConfig)
+	sess, err := session.NewSession()
```

Comment:
This is good - but did staticcheck pick this up?  And if so, how?  I tried bringing these packages into validation and didn't get any errors.